examples/plot_fpca_inverse_transform_outl_detection.py

```diff
@@ -0,0 +1,231 @@
+"""
+Outlier detection with FPCA
+===========================
+
+Example of using the inverse_transform method
+in the FPCA class to detect outlier(s) from
+the reconstruction (truncation) error.
+
+In this example, we illustrate the utility of the inverse_transform method
+of the FPCA class to perform functional outlier detection.
+Roughly speaking, an outlier is a sample
+which is not representative of the dataset
+or different enough compared to a large part of the samples.
+The intuition is the following: if the eigen basis,
```

Comment:
I think eigenbasis is one word:
```suggestion
The intuition is the following: if the eigenbasis,
```